release-tools/src/main/java/org/springframework/data/release/jira/JiraIssue.java

```diff
@@ -37,36 +47,168 @@
 	/**
 	 * Returns whether the ticket is only fixed in the given {@link Train}. {@literal false} indicates the {@link Ticket}
 	 * has been resolved for
-	 * 
+	 *
 	 * @param train
 	 * @return
 	 */
 	public boolean wasBackportedFrom(Train train) {
 
 		List<FixVersions> fixVersions = fields.getFixVersions();
-		return !(fixVersions.size() == 1 && fixVersions.get(0).name.contains(train.getName()));
+		return fixVersions != null && !(fixVersions.size() == 1 && fixVersions.get(0).name.contains(train.getName()));
```

Comment:
Can we pull the second part of the comparison into a method into an instance method?